vm/tests/solutions/spec/apache2/stackdriver_spec.rb

```diff
@@ -0,0 +1,41 @@
+# Copyright 2019 Google LLC
```

Comment:
@armandomiani @vcanaa FYI: It should be link instead pointed to the https://github.com/GoogleCloudPlatform/click-to-deploy/blob/f23b82c1bad44bba8f6b0b979638cd7a7e0c8ccb/vm/tests/solutions/spec/__common__/stackdriver_installed.rb.

Reply:
Being handled on PR #985